### ISSUE TITLE
logits and labels must be broadcastable

### ISSUE THREAD

**Issue by Satyajit1993** [krishnaik06/Deep-Learning-Car-Brand#5]
Respected sir

I am using this code to my custom dataset which is having 7 classes. While doing model.fit, i am getting this error

logits and labels must be broadcastable: logits_size=[32,8] labels_size=[32,7]

In model.summary las dense layer it is showing

dense (Dense)                   (None, 8)            802824      flatten[0][0]  

How to solve it sir?

**Comment by Satyajit1993** [comment]
desktop.ini virus was counted as a class so 7+1 =8. After a complete scan i t works fine.
